src/dlgIRC.cpp

```diff
@@ -753,6 +758,18 @@ int dlgIRC::readIrcHostPort(Host* pH)
     return port;
 }
 
+bool dlgIRC::readIrcHostSecure(Host* pH)
+{
+    QString secureStr = pH->readProfileData(dlgIRC::HostSecureCfgItem);
+    bool secure;
+    if (secureStr.toInt()) {
+        secure = true;
+    } else {
+        secure = false;
+    }
+    return secure;
```

Comment:
There is no need for the `bool secure` intermediate - the return value can be more concisely generated by:
```suggestion
    return secureStr.toInt();
```